questions/tests/test_get_next_question.py

```diff
@@ -0,0 +1,77 @@
+from django.test import TestCase
+
+from django.db import connection
+from django.test.utils import override_settings
+from emailusername.models import User
+
+from questions import models
+from questions.views import _get_next_question
+from questions.views import NextQuestion
+
+
+class TestGetNextQuestion(TestCase):
+
+    def setUp(self):
+        # Create a user
+        self.PASSWORD = 'p'
+        self.USERNAME = 'foo@bar.com'
+        self.user = User(email=self.USERNAME)
+        self.user.set_password(self.PASSWORD)
+        self.user.save()
+
+    def _assign_question_to_user(self, user, question):
```

Comment:
Hmm right, although what the method really accomplishes is linking a question to a user via `Tags`, which are basically relationships between questions and users with some metadata, correct?

What about a name like `_associate_question_with_user`?